doozer/doozerlib/cli/images_streams.py

```diff
@@ -738,7 +738,7 @@ def search_issues(query):
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
             'versions': [{'name': release_version}],  # Affects Version/s
-            'customfield_12323140': [{'name': Model(runtime.gitdata.load_data(key='bug').data).jira_config.target_release[-1]}],  # customfield_12323140 is Target Version in jira
+            'customfield_12323140': [{'name': runtime.gitdata.load_data(key='bug').data.get('target_release')[-1]}],  # customfield_12323140 is Target Version in jira
```

Comment:
Properties under `jira_config` need to be merged into the parent object. See https://github.com/openshift/elliott/blob/18aef26c32b01bebd2983871467c75927ab43b9a/elliottlib/bzutil.py#L583

Reply:
what do you mean? target_release is not under jira_config

Reply:
Please read the code I linked. Properties under `jira_config` (including `target_release`) are merged into the top-level object of `bug.yml`.

Reply:
That means target_release can be a field of either jira_config or the top level object.